### PR TITLE
Bug 1219627 - Implement test_browser_play_video as a Gij test r=mwargers

### PR DESCRIPTION
**Pull request by albertopq**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=77d9d859c8b0879ec386435217ec5f9dcbf06c23

File: apps/system/test/marionette/video_test.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    player.invokeControls();
+    player.tapFullscreen();
+
+    client.switchToFrame();
+    client.waitFor(function() {
+      return system.permissionDialog.displayed();
+    });
+
+    system.permissionYes.click();
```
Comment (mwargers) [inline apps/system/test/marionette/video_test.js]: Why is this called click(). Shouldn't we have to tap on this button?

File: apps/system/test/marionette/video_test.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+'use strict';
```
Comment (mwargers) [inline apps/system/test/marionette/video_test.js]: This file is part of the system app? Shouldn't it be part of the search app?
The search app already has a browser test there? http://mxr.mozilla.org/gaia/source/apps/search/test/marionette/browser_test.js
Also, perhaps this file should be called browser_video_test.js, to make sure that it's not mistaken for a video_player test.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6dd6037a37efc1c5f04ba3c031f3c6b50b6cb191

File: apps/system/test/marionette/video_test.js (hunk 2 above)
Comment (albertopq) [inline apps/system/test/marionette/video_test.js]: In gaia, the browser is just a special window in the system. There is not different app for the browser (despite we show an icon), and that's the reason all the browser tests are inside the system. We could potentially group them in a folder for make it more understandable. +1 to the renaming

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=dc53adf01b563366260ea9d46295894890029dcd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4d547144eead7d492123e6a6335dbe9090dc27a4

**Pull request merged**